.github/workflows/docpr.yml

```diff
@@ -0,0 +1,40 @@
+name: AutoDocumentPR
+
+on:
+  push:
+    branches: 
+      - dev
+      - master
+
+jobs:
+  dokka:
+    name: Dokka
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v1
+
+      - name: "Cache ~/.gradle/caches"
+        uses: actions/cache@preview
+        with:
+          path: "~/.gradle/caches"
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/build.gradle.kts') }}
+          restore-keys: ${{ runner.os }}-gradle-
+
+      - name: Setup JDK
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+          
+      - name: Run dokka
+        run: |
+          rm -rf dokka
```

Comment:
what is this supposed to delete? There is no directory by the name dokka. Dokka is just a gradle action